index.js

```diff
@@ -217,13 +220,22 @@ const captureWebsite = async (url, options) => {
 	if (!options._keepAlive) {
 		await browser.close();
 	}
+	// If device frame is true, we need to add device frame here.
 
-	return buffer;
+	const dfConfig = framesConfig[options.emulateDevice];
```

Comment:
It should throw as useful if you try to use a device without a frame.

Reply:
please suggest what changes needed.

Reply:
I’m confused. I already did.

Reply:
Sorry, I am unable to understand what you mean by `It should throw`? If you mean `throw new Error` then It is done now.

Reply:
Ah, I'm sorry, I didn't realize I wrote it incorrectly. I was on my phone. I meant to write:

> It should throw a useful error if you try to use a device without a frame.